javascript/ql/src/semmle/javascript/frameworks/ClientRequests.qll

```diff
@@ -260,6 +260,23 @@ module ClientRequest {
     }
   }
 
+  /** An expression that is used as a credential in a request. */
+  private class AuthorizationHeader extends CredentialsExpr {
+    AuthorizationHeader() {
+      exists(DataFlow::PropWrite write | write.getPropertyName() = "Authorization" |
```

Comment:
HTTP header names are case-insensitive so it might make sense to be case-insensitive here as well.

Reply:
> HTTP header names are case-insensitive so it might make sense to be case-insensitive here as well.

[new results from case-insensitivity](https://lgtm.com/query/7432913093213735323/).  
LGTM :+1: